app/services/mixpanel_analytics.rb

```diff
@@ -0,0 +1,21 @@
+module MixpanelAnalytics
+  def requested_feed(owner:, repo:)
+    event = 'Fetched feed'
+    properties = { owner: owner, repo: repo }
+    track(event, properties)
+  end
+
+  private
+
+  def track(event, properties)
+    tracker.track(user_id, event, properties)
+  end
+
+  def tracker
+    @tracker ||= Mixpanel::Tracker.new(ENV['MIXPANEL_TOKEN'])
+  end
+
+  def user_id
+    request.remote_ip
```

Comment:
What if user was the repo owner?

Think MP will figure out the IP of the request ... of that's the JS version I bet.

May be able to override the IP default in the tracking.

That way the analytics belong to the owner  -- and not if I use PullFeed at work and at home -- I'd be a different user.